### PR TITLE
IBX-4694: Added go to storefront button in welcome page

### PR DESCRIPTION
**Pull request by dew326**
**Jira ticket:** https://issues.ibexa.co/browse/IBX-4694

### CODE REVIEW

File: ibexa/commerce/4.4/templates/themes/standard/full/welcome_page.html.twig [hunk 1]
```diff
@@ -30,12 +30,11 @@
...
                     <strong>{{ ibexa_render_field(content, 'description') }}</strong>
-                    <p class="mb-0">{{ 'ibexa_platform_welcome_page.navigate_to'|trans({'%target%': project_dir})|desc('Navigate to: %target%') }}</p>
 
                     {% set admin_url = url('ibexa.dashboard', { 'siteaccess': 'admin' }) %}
-                    <p class="mb-0">
-                        {{ 'ibexa_platform_welcome_page.create_new_content_at'|trans({'%target%': '<a href="' ~ admin_url ~ '" class="text-white"><u>' ~ admin_url ~ '</u></a>' })
-                        |desc('Create new content at %target%')|raw }}
+                    <p class="mb-0 mt-3">
+                        <a href="{{ admin_url }}" class="btn btn-danger">{{ 'ibexa_platform_welcome_page.create_content'|trans|desc('Create content') }}</a>
+                        <a href="/product-catalog" class="btn btn-light ml-2">{{ 'ibexa_platform_welcome_page.go_to_storefront'|trans|desc('Go to Storefront') }}</a>
```
Comment (dew326) [inline ibexa/commerce/4.4/templates/themes/standard/full/welcome_page.html.twig:37]: I'm not sure about this href, is there a better way to add it? @webhdx @adamwojs

**Review by GrabowskiM — APPROVED** [review]

**Review by lucasOsti — APPROVED** [review]

**Comment by dew326** [comment]
@lserwatka added background to other editions as well, but I think we still have some room for improvements in this welcome page. Problably as a separate ticket.

**Review by webhdx — APPROVED** [review]
I'm not sure about the `/product-catalog` route but IMO you can leave it as is for the time being. On the default installation this won't change and even if you would like to change it you most likely have already removed welcome page in your project.

**Comment by lserwatka** [comment]
@NataliaBecla can we make sure we have a great design for those welcome pages ready for 4.4 launch?

**Review by tischsoic — APPROVED** [review]

**Comment by bogusez** [comment]
Welcome page is properly generated for all version. The storefront button is only presence for commerce and redirect user to /product-catalog page

**Review by bogusez — APPROVED** [review]

**Pull request merged**